src/test/java/com/puppycrawl/tools/checkstyle/checks/naming/LocalFinalVariableNameCheckTest.java

```diff
@@ -56,9 +58,11 @@ public void testDefault()
         final String pattern = "^[a-z][a-zA-Z0-9]*$";
 
         final String[] expected = {
-            "123:19: " + getCheckMessage(MSG_INVALID_PATTERN, "CDE", pattern),
+            "17:19: " + getCheckMessage(MSG_INVALID_PATTERN, "CDE", pattern),
```

Comment:
I moved all test inputs concerning LocalFinalVariableNameCheck which is the target class of this issue to separate file as a part of a tendency in Issue: #3891. I don't understand why it is bad, as you even have separate task on moving inputs to separate files.